src/components/config-tabs/ConfigTabs.js

```diff
@@ -10,176 +14,95 @@ import { ExternalDataComparison } from './tab-contents/ExternalDataComparison.js
-                    className={
-                        toggleState === 3 ? styles.activeTabs : styles.tabs
-                    }
-                    onClick={() => toggleTab(3)}
-                >
-                    {' '}
-                    Numerator Relations
-                </button>
-                <button
-                    className={
-                        toggleState === 4 ? styles.activeTabs : styles.tabs
-                    }
-                    onClick={() => toggleTab(4)}
-                >
-                    {' '}
-                    Numerator Quality Parameters
-                </button>
-                <button
-                    className={
-                        toggleState === 5 ? styles.activeTabs : styles.tabs
-                    }
-                    onClick={() => toggleTab(5)}
-                >
-                    {' '}
-                    Denominators
-                </button>
-                <button
-                    className={
-                        toggleState === 6 ? styles.activeTabs : styles.tabs
-                    }
-                    onClick={() => toggleTab(6)}
-                >
-                    {' '}
-                    Denominator Relations
-                </button>
-                <button
-                    className={
-                        toggleState === 7 ? styles.activeTabs : styles.tabs
-                    }
-                    onClick={() => toggleTab(7)}
-                >
-                    {' '}
-                    External Data Comparison
-                </button>
-            </div>
-
-            <div className={styles.contentTabs}>
-                {/* TODO: use useConfigurations instead of prop drilling */}
-                <div
-                    className={
-                        toggleState === 1
-                            ? styles.activeContent
-                            : styles.content
-                    }
-                >
-                    <Numerators />
-                </div>
-                <div
-                    className={
-                        toggleState === 2
-                            ? styles.activeContent
-                            : styles.content
-                    }
-                >
-                    <NumeratorGroups
-                        toggleState={toggleState}
-                        configurations={configurations}
-                    />
-                </div>
-                <div
-                    className={
-                        toggleState === 3
-                            ? styles.activeContent
-                            : styles.content
-                    }
-                >
-                    <NumeratorRelations
-                        toggleState={toggleState}
-                        configurations={configurations}
-                        mappedNumerators={mappedNumerators}
-                    />
+        <>
+            <TabBar>
```

Comment:
ideally this would be `scrollable`, but when I add that property, I get errors (`ResizeObserver loop completed with undelivered notifications.`) These probably don't occur in production, but I'm not sure it makes sense to add for now 🤔?